.github/workflows/application-onboarding-helm-template.yml

```diff
@@ -0,0 +1,98 @@
+        uses: actions/checkout@v2
+        with:
+          repository: department-of-veterans-affairs/vsp-infra-application-manifests
+          token: ${{ secrets.VA_VSP_BOT_GITHUB_TOKEN }}
+          path: vsp-infra-application-manifests
+
+      - name: Check out Helm template infra repo
+        uses: actions/checkout@v2
+        with:
+          repository: department-of-veterans-affairs/vsp-infra-helm
+          token: ${{ secrets.VA_VSP_BOT_GITHUB_TOKEN }}
+          path: vsp-infra-helm
+
+      - name: Check if application directory exists
+        id: directory_check
+        run: |
+          if [ -d vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }} ]
+          then 
+            echo "::set-output name=directory_exists::$(echo true)"
+          else
+            echo "::set-output name=directory_exists::$(echo false)"
+          fi
+
+      - name: Create application directory
+        if: steps.directory_check.outputs.directory_exists == 'false'
+        run: mkdir -p vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}
+
+      - name: Copy Chart.yaml from template
+        if: steps.directory_check.outputs.directory_exists == 'false'
+        run: cp vsp-infra-helm/hello-world/Chart.yaml vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/Chart.yaml
+
+      - name: Copy values.yaml from template
+        if: steps.directory_check.outputs.directory_exists == 'false'
+        run: cp vsp-infra-helm/hello-world/values.yaml vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
+
+      - name: Modify image to pull from ECR
+        if: steps.directory_check.outputs.directory_exists == 'false'
+        run: cat <<< $(yq -e '(.image.repository)="${{ secrets.HTTPBIN_ECR_REPOSITORY}}"' vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml) > vsp-infra-application-manifests/apps/${{ inputs.application_name }}/${{ inputs.application_environment }}/values.yaml
+
+      - name: Modify ingress environment to use dev
+        if: steps.directory_check.outputs.directory_exists == 'false'
+        run: cat <<< $(yq -e '(.ingress.environment)="${{ inputs.application_environment }}"' vsp-infra-application-manifests/apps/om-test/dev/values.yaml) > vsp-infra-application-manifests/apps/om-test/dev/values.yaml
+
+      - name: Create pull request
+        if: steps.directory_check.outputs.directory_exists == 'false
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          path: vsp-infra-application-manifests
+          token: ${{ secrets.VA_VSP_BOT_GITHUB_TOKEN }}
+          branch: "chore/${{ inputs.task }}-${{ inputs.application_name }}-${{ inputs.application_environment }}"
```

Comment:
As of now it is creating a branch, but should I just have it commit to `main` directly?

Reply:
Honestly I see no problem with it committing directly once we trust the automation.

Reply:
Ayeee. Would like to do final run before changing the branch